lib/SIL/IR/SILPrinter.cpp

```diff
@@ -3630,6 +3635,15 @@ void SILFunction::print(SILPrintContext &PrintCtx) const {
     OS << "[available " << availability.getVersionString() << "] ";
   }
 
+  // This is here only for testing purposes.
```

Comment:
Is it just for "testing"? Seems like would be useful to just keep around anyway right?